packages/@ember/-internals/glimmer/lib/helpers/-parse-int.ts

```diff
@@ -0,0 +1,11 @@
+import { Arguments, VM } from '@glimmer/runtime';
+import { InternalHelperReference } from '../utils/references';
+
+function parseIntHelper({ positional }: any) {
```

Comment:
```suggestion
function parseIntHelper({ positional }: CapturedArguments) {
```

You'll have to import `CapturedArguments` from `@glimmer/interfaces`